Add detail about limitation in voting contract to fix #12478 (#13387)
ethereum/solidity: docs/examples/voting.rst

Possible Improvements
=====================

Currently, many transactions are needed to

> assign the rights to vote to all participants,
> and ``winningProposal()`` will not register there
> is a tie if two or more proposals have the same
> number of votes. Can you think of a way to fix
> these issues?

Reviewer: ```suggestion
assign the rights to vote to all participants.
Moreover, if two or more proposals have the same
number of votes, ``winningProposal()`` is not able 
to register a tie. Can you think of a way to fix these issues?
```
I think this is more straightforward. What do you think?

Author: Yes I agree it's a bit easier to understand, will commit this suggestion

=====================

Currently, many transactions are needed to assign the rights
to vote to all participants, and ``winningProposal()`` will

> not work correctly if two or more proposals are tied in voteCount.

Reviewer 2: not sure I agree with the wording "will not work correctly". It's still returning "a" winning proposal

Author: Good point, will update it with something clearer
